### ISSUE TITLE
ArTaxOr_1shot Performance Issue

### ISSUE THREAD

**Issue by yikousige** [lovelyqian/CDFSOD-benchmark#55]
Hello! First of all, thank you very much for your excellent work—your paper has been incredibly inspiring to me.

I’m currently trying to reproduce the results of your code, but I’ve encountered a performance discrepancy specifically in the ArTaxOr_1shot setting. Here are the key details:

In my reproduction, the AP for ArTaxOr_1shot is around 15.48 (consistent across multiple runs).
However, this is lower than the 21.0 AP reported in your paper.
Notably, results on other settings (ArTaxOr_5shot, ArTaxOr_10shot) and other datasets match your paper closely, which suggests most of the reproduction pipeline is correct.

To help diagnose, I’ve checked basic reproducibility steps:

I used the official codebase without modifications to core logic.
Datasets were prepared strictly following the instructions in the paper (e.g., data splitting, annotation format).
I used the recommended pre-trained weights and hyperparameters (learning rate, batch size, etc.) as specified.

I wonder if you have made any special adjustments to the details of ArTaxOr_1shot?

If you have any insights or suggestions for further checks,I’d greatly appreciate your guidance.

Thank you again for your time and work!

<img width="848" height="416" alt="Image" src="https://github.com/user-attachments/assets/f1a5ea67-4a83-49f6-a270-a91e2175f78e" />

[log.txt](https://github.com/user-attachments/files/31880766/log.txt)
